docs/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+AdvancedMH = "5b7e9947-ddc0-4b3f-9b55-0d8042f74170"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
Can you add a compat entry as well and add docs/Project.toml to the CompatHelper workflow?